### PR TITLE
Improved: Subscription Management: Show the position number

### PR DESCRIPTION
**Pull request by math-GH**
The issue: it is a bit tricky to order the categories. The position number is more or less invisible.

The position is visible only in the settings:
grafik

Now the number is shown in the box
grafik

Changes proposed in this pull request:

- PHTML and CSS

How to test the feature manually:

1. go to subscription management
2. config the position number
3. see the number on the box

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by math-GH** [comment]
Maybe someone could support here:

These lines are not the best in the template:
```
$categoryDAO = FreshRSS_Factory::createCategoryDao();
$category = $categoryDAO->searchById($cat->id());
$pos = $category->attributes('position');
```

**Comment by aledeg** [comment]
> Maybe someone could support here:
> 
> These lines are not the best in the template:
> 
> ```
> $categoryDAO = FreshRSS_Factory::createCategoryDao();
> $category = $categoryDAO->searchById($cat->id());
> $pos = $category->attributes('position');
> ```

I think (not tested) that you could drop that and instead of `$pos` use `$cat->attributes('position')` directly in the template. You are already in a loop where you are using categories.

File: app/views/subscription/index.phtml [hunk 1]
```diff
@@ -33,8 +33,13 @@
 				$feeds = $cat->feeds();
 		?>
 		<div class="box">
+			<?php
+				$categoryDAO = FreshRSS_Factory::createCategoryDao();
+				$category = $categoryDAO->searchById($cat->id());
```
Comment (Alkarex) [inline app/views/subscription/index.phtml]: We probably already have all the categories in our context. See also `$this->categories` and its definition in:

https://github.com/FreshRSS/FreshRSS/blob/3b54f8cd8196728df14590bfb1560dd4f6aaf685/app/Controllers/subscriptionController.php#L22

Ups, by the way, I can see that it looks like we have one `true` too many

Comment (math-GH) [reply app/views/subscription/index.phtml]: I fixed the doubled parameter

**Comment by math-GH** [comment]
Thanks for your help. It works :)

**Comment by aledeg** [comment]
@math-GH There is an issue when the category name is really long. The number and the name are entangled. It's quite impossible to see it. It might be a good idea to add something to handle that case.

**Pull request merged**